test/cypress/integration/request-to-go-live/organisation_name_spec.js

```diff
@@ -80,11 +74,29 @@ describe('Request to go live: organisation name page', () => {
     })
   })
 
-  describe('Service has correct go live stage and organisation name is not pre-filled', () => {
+  describe('Service has invalid go live stage', () => {
+    const serviceRole = utils.buildServiceRoleForGoLiveStage('INVALID_GO_LIVE_STAGE')
+    beforeEach(() => {
+      utils.setupStubs(serviceRole)
+    })
+    it('should redirect to "Request to go live: index" page when in wrong stage', () => {
+      const requestToGoLivePageOrganisationNameUrl = `/service/${serviceExternalId}/request-to-go-live/organisation-name`
+      cy.visit(requestToGoLivePageOrganisationNameUrl)
+
+      cy.get('h1').should('contain', 'Request to go live')
+
+      cy.location().should((location) => {
+        expect(location.pathname).to.eq(`/service/${serviceExternalId}/request-to-go-live`)
+      })
+    })
+  })
+
+  describe('Service has NOT_STARTED go live stage and organisation name is not pre-filled', () => {
+    const serviceRoleBeforePatchRequests = utils.buildServiceRoleForGoLiveStage('NOT_STARTED')
+    const serviceRoleAfterPatchRequests = utils.buildServiceRoleForGoLiveStage('ENTERED_ORGANISATION_NAME')
```

Comment:
I like the naming of these variables, I think it makes the confusing thing we are doing with the different stub responses a lot clearer 👍